src/VirtoCommerce.NotificationsModule.Data/Migrations/20190528144557_InitialNotifications.cs

```diff
@@ -322,16 +350,46 @@ INNER JOIN
     			                            THEN 'SmsNotificationMessageEntity'
     			                            ELSE 'EmailNotificationMessageEntity'
     		                            END
-	                            END
+	                            END [Discriminator]
 	                            FROM [PlatformNotification] pn
-	                            INNER JOIN 
+	                            INNER JOIN pnt2

```

Comment:
are there notification messages without NotificationId? Is it ok?
![image](https://user-images.githubusercontent.com/8361000/97417088-f0cb6f00-190f-11eb-95cd-b758148608d8.png)

Reply:
Yes, they are.

Reply:
did you check opening details in notification feeds?

Reply:
Yes, I did.